lib/bundler.js

```diff
@@ -8,47 +8,51 @@ var cloneDeep = require('lodash').cloneDeep;
  * @param {Object} instructions Boot instructions.
  * @param {Object} bundler A browserify object created by `browserify()`.
  */
-
-module.exports = function addInstructionsToBrowserify(instructions, bundler) {
-  bundleModelScripts(instructions, bundler);
-  bundleMixinScripts(instructions, bundler);
-  bundleComponentScripts(instructions, bundler);
-  bundleOtherScripts(instructions, bundler);
-  bundleInstructions(instructions, bundler);
+module.exports = function addInstructionsToBrowserify(context, bundler) {
+  // bundlePluginScripts(context, bundler);
```

Comment:
I think it's crucial to bundle plugin scripts too, otherwise we may not have the code to run during the `execute` phase in the browser. Am I missing something?

Also to keep the footprint of the browser bundle small, we need a mechanism that will allow plugin authors to split their code into two files: one providing `compile` part that will be run while browserifying, the other one providing `execute` part that will be included in the browser bundle.